language-server/modules/langserver-core/src/test/resources/completion/expression_context/source/list_constructor_ctx_source5.bal

```diff
@@ -0,0 +1,4 @@
+function testListConstructorSpreadOp() {
+    int[] x = [1, 2];
+    int[] y = [...];
```

Comment:
We have to fix the sorting of the completions at the cursor position after `...`. Further, we may have to add a spread op completion item as well (Just like spread field completion item for mapping constructors). Will create a separate issue for this.

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/35507 tracks the changes to be made